renovate.json5

```diff
@@ -5,27 +5,24 @@
   dependencyDashboard: true,
   packageRules: [
     {
-      // Those cannot be upgraded to a major version until we drop support for Node 10
+      // Those cannot be upgraded to a major version until we switch to ESM
       packageNames: [
-        'path-type',
-        'open',
-        'inquirer',
-        'p-timeout',
-        'configstore',
-        'p-wait-for',
-        'path-key',
-        'strip-ansi',
         '@sindresorhus/slugify',
         'ansi-styles',
-        'is-plain-obj',
-        'log-symbols',
-        'random-item',
         'better-opn',
-        'to-readable-stream',
+        'configstore',
         'http-proxy-middleware',
```

Comment:
I discovered that this package can be updated since Node.js 12 is no longer supported and doesn't have any other breaking changes.